lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterBase.php

```diff
@@ -117,7 +117,7 @@ protected function dispatchMany(array $nodes): void
      *
      * @return object|array - PHPCR QOM object or array of objects
      */
-    public function dispatch(AbstractNode $node)
+    public function dispatch(QBConstants $node)
```

Comment:
`AbstractNode` seems to be in the same namespace. Neither `use` is needed nor the alias.

Reply:
that was the cs fixer. i think someone originally aliased AbstractNode to QBConstants for using the constants, while keeping AbstractNode as typehint... changed it all to AbstractNode now.